Android/app/src/main/java/com/example/android/models/MenuItem.java

```diff
@@ -1,10 +1,17 @@
 package com.example.android.models;
 
-public class MenuItem {
+import java.io.Serializable;
+
+public class MenuItem implements Serializable {
     private String photoUrl;
     private String name;
     private Double price;
     private boolean available;
+    private String description;
+
+    public String getDescription() {
```

Comment:
When you are doing getters and setters use annotations @Gettet and @Setter also for args and noargs constructor